metsim/metsim.py

```diff
@@ -539,6 +550,12 @@ def _validate_setup(self):
         if not len(self.params.get('forcing', [])):
             errs.append("Requires input forcings to be specified")
 
+        # Make sure there is at least one forcing_var
+        # They cannot all be constant since we use one as a template
+        # for the others
+        if not len(self.params.get('forcing_vars', [])):
+            errs.append("Requires at least one non-constant forcing")
```

Comment:
This error never gets "exercised", because `_self_validate()` is executed *AFTER* the met data are read in. We should probably change that a little so we can check the setup *BEFORE* reading the met data (see also issue #169)